### PR TITLE
feat: Formatting code/comments, updating tests (SC-7580)

### PR DESCRIPTION
**Pull request by lucas-manuel**

### CODE REVIEW

File: src/Univ2LpOracle.sol [hunk 1]
```diff
@@ -204,38 +202,30 @@ contract UNIV2LPOracle {
         return block.timestamp >= add(zzz, hop);
     }
 
-    function seek() public returns (uint128 quote, uint32 ts) {
-        //sync up reserves of uniswap liquidity pool
+    function seek() public returns (uint128 quote, uint32 ts) { // Why do we pass in ts here?
```
[the remark this answers is not in the document]
Comment (NiklasKunkel) [reply src/Univ2LpOracle.sol]: @lucas-manuel Can you elaborate on what you mean by "if `poke()` is called in the next block"? Poke only gets called every period `pass` and will do the call to seek. Are you saying that doing a `block.timestamp` is a SREAD, so doing it both in `seek()` and again in `poke()` would waste more gas than just MSTORE the result of `block.timestamp` in `seek()` to memory and doing the second MSTORE+MREAD in `poke()`. Apologies if that sounds convoluted, a bit hard to describe.

[the remark this answers is not in the document]
Comment (lucas-manuel) [reply src/Univ2LpOracle.sol]: Ooh sorry I didn't see the call to `seek()` in `poke()`. Knowing that I'll consider the best gas approach and push.

[the remark this answers is not in the document]
Comment (lucas-manuel) [reply src/Univ2LpOracle.sol]: Yeah now that I understand how that's working, I think it's fine to leave as is.

**Comment by lucas-manuel** [comment]
@NiklasKunkel @brianmcmichael this is ready for final review.

File: src/Univ2LpOracle.sol [hunk 2]
```diff
@@ -127,7 +152,7 @@ contract UNIV2LPOracle {
     function wdiv(uint x, uint y) internal pure returns (uint z) {
         z = add(mul(x, WAD), y / 2) / y;
     }
-    //compute square using babylonian method
+    // Compute square using babylonian method
```
Comment (kmbarry1) [inline src/Univ2LpOracle.sol:155]:
```suggestion
    // Compute the square root using the Babylonian method.
```

**Pull request merged**